dockerfiles/configure-qpl

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
```

Comment:
```suggestion
#!/usr/bin/sh
```
nit

Reply:
I get way more hits for `#!/bin/sh` than `#!/usr/bin/sh` on github search. I have no idea when/why to prefer one or the other. Can you tell me why you'd use `/usr`?

Reply:
Had this confused with `/usr/bin/env bash`.
Other than `/bin` being a symlink to `/usr/bin`, there is no real difference.

Thinking about it again, we probably don't want to use `/usr/bin/env bash` since we only execute the script in an environment where we know the real path to bash